rs/nervous_system/integration_tests/tests/swap_node_in_subnet_directly.rs

```diff
@@ -0,0 +1,37 @@
+use ic_base_types::PrincipalId;
+use ic_nervous_system_integration_tests::pocket_ic_helpers::{self, NnsInstaller};
+use pocket_ic::PocketIcBuilder;
+use registry_canister::mutations::do_swap_node_in_subnet_directly::{
+    SwapError, SwapNodeInSubnetDirectlyPayload,
+};
+
+// This test ensures that we are not enabling this feature on any network until it
+// is fully implemented.
+//
+// TODO(DRE-551): adapt the logic of the test to not fail if the feature is enabled.
+#[tokio::test]
+async fn ensure_feature_is_turned_off() {
+    let pocket_ic = PocketIcBuilder::new()
+        .with_nns_subnet()
+        .with_sns_subnet()
+        .build_async()
+        .await;
+
+    let mut nns_installer = NnsInstaller::default();
+    nns_installer.with_current_nns_canister_versions();
+    nns_installer.install(&pocket_ic).await;
```

Comment:
This is mild overkill for a test of just the registry canister, and will make your test slower.  You only need Registry (and sometimes Governance).

Reply:
Totally agree, In the new version I've added the installation just for the registry canister so that we save time and resources. Thanks!